conditionalSwitch.js

```diff
@@ -0,0 +1,14 @@
+function getLetter(s) {
+    let letter;
```

Comment:
why you have kept letter like this. you should know that in this way letter is undefined